pkg/accounts/mod.ts

```diff
@@ -127,17 +127,17 @@ const verifyAccountTokenService = Cat.cat(verifyAccountToken)
   .feed(Ether.compose(accountRepository)).value;
 
 const composer = Ether.composeT(Promise.monad);
-const liftOverPromise = <const D extends Record<string, symbol>, T>(
-  ether: Ether.Ether<D, T>,
-): Ether.EtherT<D, Promise.PromiseHkt, T> => ({
-  ...ether,
-  handler: (resolved) => Promise.pure(ether.handler(resolved)),
-});
+const liftOverPromise = Ether.liftEther(Promise.monad);
+
+const authToken = await Cat.cat(authenticateToken).feed(
+  composer(liftOverPromise(clock)),
+).value;
```

Comment:
```suggestion
const authToken = Cat.cat(authenticateToken).feed(
  composer(liftOverPromise(clock)),
).value;
```
この `await` は効果がないです. この `.value` は `Promise` 上で動くようにラップされた `Ether` なので, これ自体は `Promise` ではありません. 他のパッケージの `mod.ts` も同様です.